huaweicloud/services/dms/resource_huaweicloud_dms_kafka_smart_connect_task.go

```diff
@@ -0,0 +1,327 @@
+package dms
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// API: DMS POST /v2/{project_id}/connectors/{connector_id}/sink-tasks
```

Comment:
the product should be **Kafka**
```suggestion
// API: Kafka POST /v2/{project_id}/connectors/{connector_id}/sink-tasks
```

Reply:
done